Xwt.Gtk/Xwt.GtkBackend/ButtonBackend.cs

```diff
@@ -272,6 +307,15 @@ void HandleSizeAllocated (object o, Gtk.SizeAllocatedArgs args)
 			Widget.Child.SizeAllocate (args.Allocation);
 			args.RetVal = true;
 		}
+
+		protected override void Dispose (bool disposing)
+		{
+			if (labelWidget != null) {
```

Comment:
```suggestion
			if (disposing && labelWidget != null) {
```
and set `labelWidget = null` at the end of the block to not NRE on double disposing.